assets/js/sidebar.js

```diff
@@ -49,7 +62,9 @@ function openSidebar (immediate) {
     '-ms-flex-basis': width,
     'flex-basis': width,
     width: width
-  }, duration)
+  }, duration, function () {
+    sidebar.attr('style', '')
```

Comment:
I think this line introduced a bug where the sidebar never really toggles for small screens. Why do we need it?